modules/markup/sanitizer.go

```diff
@@ -58,6 +58,12 @@ func createDefaultPolicy() *bluemonday.Policy {
 	// For color preview
 	policy.AllowAttrs("class").Matching(regexp.MustCompile(`^color-preview$`)).OnElements("span")
 
+	// For attention
+	policy.AllowAttrs("class").Matching(regexp.MustCompile(`^attention-\w+$`)).OnElements("span", "strong")
+	policy.AllowAttrs("class").Matching(regexp.MustCompile(`^svg octicon-\w+$`)).OnElements("svg")
+	policy.AllowAttrs("viewBox", "width", "height", "aria-hidden").OnElements("svg")
+	policy.AllowAttrs("fill-rule", "d").OnElements("path")
```

Comment:
`fill`, `stroke` and `clip-path` are more properties that may appear on some of our SVGs.

Reply:
Yeah they are only in some of the logos.